### PR TITLE
Add EM career ladder.

### PR DESCRIPTION
**Pull request by mathias-abtion**
As we've talked about a few times now, we need an EM career ladder. This pull request provides a draft of what such a ladder might look like.

For now, I've mostly focused on level 3, but if it makes sense, we could consider moving some of the points into later levels. Most larger corporations seem to divide these levels into Engineering Manager, Senior Engineering Manager, Head of Engineering, Director of Engineering, and Vice President of Engineering. I am not sure these titles make sense in a company of our size, though, but let me know what you think.

Also, I've noted it as a prerequisite that you are at least on level 2 in the existing IC career ladder since I think it is quite important that you have sharpened your technical skills before you move into a role as an Engineering Manager. At the same time, though, I also want to acknowledge the fact that EMs are not ICs, and the expectations, therefore, are different for the two tracks.

@hcarreras: Are EMs responsible for managing contract changes? Who puts their name on contracts etc.? Is this something we want to cover in the career ladder?

### CODE REVIEW

File: career/em-ladder.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Engineering Management Career Ladder
+This career ladder serves to give an overview of how you can progress your
+career at Abtion and become an Engineering Manager.
+
+As a prerequisite, we require people to have moved beyond level 2 on our [IC
```
[the remark this answers is not in the document]
Comment (MarkusAbtion) [reply career/em-ladder.md:5]: IC stands for Individual contributor

File: career/em-ladder.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+# Engineering Management Career Ladder
+This career ladder serves to give an overview of how you can progress your
+career at Abtion and become an Engineering Manager.
```
[the remark this answers is not in the document]
Comment (mathias-abtion) [reply career/em-ladder.md:3]: Very good suggestion. Thanks.

File: career/em-ladder.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mathias-abtion) [reply career/em-ladder.md:5]: I'll add it to the document so both "Individual contributor" and "IC" are present in the document.

**Comment by mathias-abtion** [comment]
There is still plenty of room for improving the ladder but I am going to merge it now. Let's keep refining it in separate pull requests instead.

**Review by hcarreras — COMMENTED** [review]
I'm loving the new points that you added. Excellent work, Mathias 👏

**Pull request merged**